.github/workflows/main.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       run: sudo -E make test
 
     - name: Test magicgen script
-      run: sudo -E make -C scripts test
+      run: make -C scripts/magic-gen test
```

Comment:
@snprajwal Could we also run this test within the [`coverage`](https://github.com/checkpoint-restore/go-criu/blob/83fc1377679b412d169473ed1d58b47a3cf385d7/test/Makefile#L77) target?

Reply:
Yes, I'll add that there in the next push